pages/common/rustfmt.md

```diff
@@ -0,0 +1,15 @@
+# rustfmt
+
+> Tool for formatting Rust source code.
+
+- Format a file, overwriting the original file in-place:
+
+`rustfmt {{source.rs}}`
+
+- Check a file for formating and display any changes on console:
+
+`rustfmt --check {{source.rs}}`
+
+- Backup any modified files before formatting:
```

Comment:
What does it backup the file to ? sth like source.rs.bak ?

Reply:
The backup file for `source.rs` -> `source.bk` by default.

Reply:
Let's mention that in the description.

Reply:
sure @agnivade